org.sofproject.alsa.topo/META-INF/MANIFEST.MF

```diff
@@ -10,3 +10,4 @@ Export-Package: org.sofproject.alsa.topo.conf,
 Require-Bundle: org.sofproject.core,
  org.sofproject.topo.ui
 Bundle-Vendor: Intel Corporation
+Import-Package: org.sofproject.gst.json
```

Comment:
alsa.topo and gst are independent plug-ins, potentially used separately. There should be no dependencies introduced between them.

If you want to make it common for all topology plug-ins, and serialize through public interface of topology elements, it should be implemented inside topo.ui plug-in and communicate with gst.topo/alsa.topo like other elements of topo.ui do.